.buildkite/steps/test-demo-notebooks.sh

```diff
@@ -33,10 +33,17 @@ f=${NOTEBOOKS[$INDEX]}
 
 case $(basename "$f") in
   'attacks_clustering_analysis.ipynb' | 'hateful-twitters-interpretability.ipynb' | 'hateful-twitters.ipynb' | 'stellargraph-attri2vec-DBLP.ipynb' | \
-    'node-link-importance-demo-gat.ipynb' | 'node-link-importance-demo-gcn.ipynb' | 'node-link-importance-demo-gcn-sparse.ipynb' | 'rgcn-aifb-node-classification-example.ipynb')
+    'node-link-importance-demo-gat.ipynb' | 'node-link-importance-demo-gcn.ipynb' | 'node-link-importance-demo-gcn-sparse.ipynb' | 'rgcn-aifb-node-classification-example.ipynb' | \
+    'stellargraph-metapath2vec.ipynb' | 'gcn-lstm-LA.ipynb')
     # These notebooks do not yet work on CI:
     # FIXME #818: datasets can't be downloaded
     # FIXME #819: out-of-memory
+    # FIXME #849: CI does not have neo4j
+    # FIXME #907: socialcomputing.asu.edu is down
+    # FIXME #1303: METR_LA dataset can't be downloaded automatically
+    # FIXME #818: datasets can't be downloaded
+    # FIXME #819: out-of-memory
```

Comment:
This has added a whole pile more lines then it needs to: the #849 and #907 fixme's have been resurrected and the #818 and #819 ones have been duplicated
```suggestion
    # FIXME #1303: METR_LA dataset can't be downloaded automatically
```